tests/Aspire.StackExchange.Redis.Tests/RedisContainerFixture.cs

```diff
@@ -34,7 +34,7 @@ public async Task DisposeAsync()
     public static async Task<RedisContainer> CreateContainerAsync()
     {
         var container = new RedisBuilder()
-            .WithImage($"{TestConstants.AspireTestContainerRegistry}/{RedisContainerImageTags.Image}:{RedisContainerImageTags.Tag}")
+            .WithImage($"{ComponentTestConstants.AspireTestContainerRegistry}/{RedisContainerImageTags.Image}:{RedisContainerImageTags.Tag}")
```

Comment:
Unrelated?

Reply:
I added a TestConstants type and renamed this one to avoid conflict.